cmd/plugin.go

```diff
@@ -294,6 +302,14 @@ func runPluginInstallCmd(cmd *cobra.Command, args []string) {
 
 		statushooks.Done(ctx)
 	}
+	// if --default-config is set to false
+	if !defaultConfig {
```

Comment:
This is not the right way to solve this issue.

Consider this scenario:
1. steampipe plugin install aws - (aws.spc file gets created)
2. configure aws.spc with creds and query
3. steampipe plugin uninstall aws - (plugin gets uninstalled, but aws.spc stays on)
4. steampipe plugin install aws ----default-config=false - (would delete the existing aws.spc config)

We should not remove the config but rather prevent its creation in the first place.